### PR TITLE
Docs/clarify configuration tools

### PR DESCRIPTION
**Pull request by didier-wenzek**
## Proposed changes

- Move the whole content of the *Configuration Tools* page to the *Configuration* index page 
- Move the example to the *Configuration* index page
- Remove unnecessarily repeated content 

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue

https://github.com/thin-edge/thin-edge.io/issues/2641

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/src/operate/configuration/index.md [hunk 1]
```diff
@@ -5,8 +5,171 @@ sidebar_position: 2
...
+
+### Reset a configuration value to use default value
+
+Unset any user-specific value for the `c8y.url` setting, using then the default value.
+
+```sh
+tedge config unset c8y.url
+```
+
+## Example
```
Comment (reubenmiller) [inline docs/src/operate/configuration/index.md:60]: Make example a subsection so it is easier to extend in the future, and it is easier for users to find via the page's TOC

```suggestion
### Examples

### Change path used for temporary files
```

**Review by reubenmiller — APPROVED** [review]
Nice improvement. Approved

File: docs/src/operate/configuration/index.md [hunk 2]
```diff
@@ -5,8 +5,171 @@ sidebar_position: 2
...
+
+### Reset a configuration value to use default value
+
+Unset any user-specific value for the `c8y.url` setting, using then the default value.
+
+```sh
+tedge config unset c8y.url
+```
+
+### Examples
```
Comment (reubenmiller) [inline docs/src/operate/configuration/index.md:60]:
```suggestion
## Example

### Change path used for temporary files
```

Comment (didier-wenzek) [reply docs/src/operate/configuration/index.md]: Oops! Fixed.

**Pull request merged**